### ISSUE TITLE
Missing Data & Wifi tiles

### ISSUE THREAD

**Issue by Zilla0617** [siavash79/AOSPMods#391]
### Describe the issue

Separate tiles, wifi and mobile data missing after update to Canary-152

### Steps to reproduce

Separate tiles for data and wifi missing after turning on switch to separate tiles on canary 152 

### Expected bahavior

Separate tiles to be listed after switching on the option 

### Your device setup (Please take some time to give correct information here)

Google Pixel 6 pro stock firmware October Security Patch
Android 13 Stable
October Security Patch
Magisk 25.2 stable
![Screenshot_20221015-102839](https://user-images.githubusercontent.com/71089409/195992962-5572eecf-fe37-4ccc-8d44-847a3c7d7b68.png)
![Screenshot_20221015-102901](https://user-images.githubusercontent.com/71089409/195992975-35eb32d1-2348-4017-9a32-a3b2d6d1ce99.png)
LSposed modules: call recording, AOSP mods, classic power menu, project themer, xdowngrader, no storage restrict

![Screenshot_20221015-110820](https://user-images.githubusercontent.com/71089409/195993773-e72285dd-3a92-475e-b695-61b06620b703.png)

LSposed 1.8.4

**Comment by Zilla0617** [comment]
![Screenshot_20221015-113110.png](https://user-images.githubusercontent.com/71089409/195994857-a16fb5a4-7380-409c-be3a-1299ff12e553.png)

**Comment by Zilla0617** [comment]
[LSPosed_2022-10-15T14_36_00.156436.zip](https://github.com/siavash79/AOSPMods/files/9792952/LSPosed_2022-10-15T14_36_00.156436.zip)
[LSPosed_2022-10-15T14_35_56.318392.zip](https://github.com/siavash79/AOSPMods/files/9792954/LSPosed_2022-10-15T14_35_56.318392.zip)

**Comment by EleoXDA** [comment]
Please repeat log taking after following steps:
- disable all magisk modules in magisk manager (other than AOSPMods and LSposed)
- disable all LSposed modules in LSposed (other than AOSPMods)
- Extract your current setup in AOSPMods and store it in your SD card
- Reboot your device
- Boot and replicate the issue
- If you can successfully reproduce issue, extract logs again with same steps and attach here.
- If you cannot successfully reproduce issue, the problem is in one of your other modules (as it was with face unlock before)

**Comment by Zilla0617** [comment]
Issue still present logs attached

Oct 15, 2022 5:10:10 PM Elbay Malik ***@***.***>:

> Please repeat log taking after following steps:
> 
* > disable all magisk modules in magisk manager (other than AOSPMods and LSposed)
* > disable all LSposed modules in LSposed (other than AOSPMods)
* > Extract your current setup in AOSPMods and store it in your SD card
* > Reboot your device
* > Boot and replicate the issue
* > If you can successfully reproduce issue, extract logs again with same steps and attach here.
* > If you cannot successfully reproduce issue, the problem is in one of your other modules (as it was with face unlock before)

**Comment by EleoXDA** [comment]
Logs are not attached. Please try to attach again. Dont reply via email. Reply INSIDE github website

**Comment by Zilla0617** [comment]
[LSPosed_2022-10-15T17_45_06.968786.zip](https://github.com/siavash79/AOSPMods/files/9793181/LSPosed_2022-10-15T17_45_06.968786.zip)

**Comment by EleoXDA** [comment]
> [LSPosed_2022-10-15T17_45_06.968786.zip](https://github.com/siavash79/AOSPMods/files/9793181/LSPosed_2022-10-15T17_45_06.968786.zip)

Please attach logs AFTER replicating the steps above. Not before those steps. You need to follow those steps + reboot with DEACTIVATED MODULES. And only after that try to reproduce

**Comment by Zilla0617** [comment]
Those are the logs after booting with disabled modules. Did you even look at the logs before emailing me

**Comment by EleoXDA** [comment]
> Those are the logs after booting with disabled modules. Did you even look at the logs before emailing me

Yes. I see all the active magisk and lsposed modules. Do follow steps correctly

**Comment by Zilla0617** [comment]
Your instructions said to disable modules, I did that. I’ll try it again for the last time

**Comment by EleoXDA** [comment]
> Your instructions said to disable modules, I did that. I’ll try it again for the last time

Currently no other user is having the issue that you describe. It is same as in your "disappearing wifi and battery icons" bugreport. All users have successfully installed and used the MOD other than you. And I continue this bugreport EXCLUSIVELY for you, not to fix a widespread issue.

So, returning to your "I’ll try it again for the last time" sentence. You can even stop trying right now. I am not asking a help from you. By following correct steps YOU make sure that YOUR issue (and again, ONLY YOU are having this issue) is getting resolved. So, if you dont want to help yourself - dont.

**Comment by Zilla0617** [comment]
I understand and appreciate it. Actually, there was an individual who reached to me who exhibited the same issue as I had with the battery indicator. They just ended up deleting your app,  I was the only one who brought it to your attention.  As far as not updating Github I completely forgot. It was months after when I found the solution, I rarely use Github.

But that’s not an excuse, I should of updated it, that’s on me.  As far the issue I’m experiencing today, both tiles appeared once I disabled all modules. I added each module one by one until all modules are enabled and both tiles are still present. Thanks for pointing me in the right direction.
